src/ccutil/ambigs.h

```diff
@@ -215,4 +217,19 @@ class UnicharAmbigs {
 
 }  // namespace tesseract
 
+
+#else
```

Comment:
The `#else` part looks like a hack which can be avoided.

Reply:
I removed it in my additional commit.

Reply:
@amitdo, do we already have more hacks like that? Then they should be examined closer, too.

Reply:
Yes, that code was meant to be temporary.

My plan was to fix it later. You already fixed it yourself. Thanks!